.vib/prestashop/cypress/cypress/integration/prestashop_spec.js

```diff
@@ -0,0 +1,130 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+// Note: Interaction with the UI is preferred over cy.visit() as
+// PrestaShop requires to provide a token within the URL or else
+// a security warning page is shown.
+
+it('allows a user to place an order and an admin to list it', () => {
+  cy.visit('/');
+  cy.get('div.product').first().click();
+  cy.contains('Add to cart').click();
+  cy.visit('/order');
+  cy.fixture('customers').then((customers) => {
+    cy.get('form#customer-form').within(() => {
```

Comment:
Why are these `within()` necessary? Are there other inputs with the same ids?

Reply:
Yes, the UI renders two forms (but only one is shown): one for registered customers and another one for guests.

Then, some of the IDs match in the hidden form and this prevents it.